bin/gxa-index-set-no-autocreate.sh

```diff
@@ -3,9 +3,9 @@ SCHEMA_VERSION=1
 
 # on developers environment export SOLR_HOST_PORT and export SOLR_COLLECTION before running
 HOST=${SOLR_HOST:-"localhost:8983"}
-CORE=${SOLR_COLLECTION:-"gxa-analytics-v$SCHEMA_VERSION"}
+CORE=${SOLR_COLLECTION:-"bulk-analytics-v$SCHEMA_VERSION"}
 
 
 curl -X POST -H 'Content-type:application/json' --data-binary '{
-  "set-user-property": {"update.autoCreateFields":"false"}
+  "set-user-property": {"update.autoCreateFields":"true"}
```

Comment:
Why do we want to change the behaviour of this script? If it is called set-no-autocreate then it seemed correct to me to be setting autoCreateFields to False instead True. Could you comment on whether this is what we want @alfonsomunozpomer ? Thanks!

Reply:
I mean, don't take my word for it, I'm not sure, that is why I was asking Alfonso. My instinct here would be not to change the script and then not use it if the suggesters require a different way of working at the testing level.

Reply:
Just to clarify on my side. The original value of autoCreateFields was originally set up to false, but this reports the error 
```
# {
#   "responseHeader":{
#     "status":400,
#     "QTime":6},
#   "error":{
#     "metadata":[
#       "error-class","org.apache.solr.common.SolrException",
#       "root-error-class","org.apache.solr.common.SolrException"],
#     "msg":"Document is missing mandatory uniqueKey field: id",
#     "code":400}}
```
Assigning a value = `true` corrected it, but it might not be doing what @alfonsomunozpomer suggested

Reply:
Based on his comments on slack, my suggestion would be that you remove that from the tests or that you make a call before it is needed to the curl call that would set that to true in the tests.

Reply:
But I wouldn't modify the behaviour of the script (it might be needed in other contests).

Reply:
That makes sense. I've created a new script `gxa-index-set-autocreate.sh` and included it in the tests